node/src/chain_spec.rs

```diff
@@ -293,6 +293,27 @@ fn testnet_genesis(
     mock_container_chains: &[ParaId],
     configuration: pallet_configuration::GenesisConfig<dancebox_runtime::Runtime>,
 ) -> dancebox_runtime::RuntimeGenesisConfig {
+    let para_ids: Vec<_> = container_chains
+        .iter()
+        .map(|x| {
+            container_chain_genesis_data_from_path(x).unwrap_or_else(|e| {
+                panic!(
+                    "Failed to build genesis data for container chain {:?}: {}",
+                    x, e
+                )
+            })
+        })
+        .chain(
+            mock_container_chains
+                .iter()
+                .map(|x| (*x, mock_container_chain_genesis_data(*x), vec![])),
+        )
+        .collect();
+    // Assign max block credits to all container chains registered in genesis
```

Comment:
If we assign max, is there any way we test running out of those? I think we should use non-max

Reply:
We test it by registering new container chains. I can set this to a high enough value so that all the tests pass, but I don't see the point.

Reply:
in dev mode its easy for us to create blocks. I think we should use a non-max, in the dev mode at least, to test this.

Reply:
lets use something in the order of 1000, we will create 1000 blocks and make sure if you run out of this then you dont get selected